### ISSUE TITLE
JOSS review

### ISSUE THREAD

**Issue by barneydobson** [IMMM-SFA/statemodify#42]
Hi @rg727 , I am a reviewer for your JOSS paper of `statemodify` and will use this issue to track things as they come up. I hope to finish this by the end of the week. Feel free to address these issues here as they come up, or create a new issue for them.

### Comments:

- [x] #43
```
FAILED statemodify/tests/test_modify.py::TestModify::test_construct_outfile_name - AssertionError: '/my/output/directory/file_S0_test.txt' != '/my/output/directory\\file_S0_test.txt'
```
- [x] #44
- [x] #47
- [x] #48
  - I see there is a `river_id` and `structure_id` in the model output file, which are the same. It seems these are the same as users? But maybe this is also the same as reservoirs? Are these four things always the same?
  - Maybe whenever you first mention a StateMod file type (e.g., `.ddr`) you could link to a description of that file type - though I can see that the description on their readthedocs is limited and when I go further in it just gets more complicated rather than clearer.. If you know where to find a good description probably link that otherwise those 'NOTE' blue boxes you have been using would be good.
- [x] #49
- [x] #50
- [x] #51
- [x] #52

openjournals/joss-reviews#6325

**Comment by crvernon** [comment]
Thanks @barneydobson , we will get on these ASAP.

**Comment by rg727** [comment]
@barneydobson thank you for your thorough review, we really appreciate getting this feedback! We will start to chip away at your suggestions ASAP!

**Comment by crvernon** [comment]
Addressed in #66
